### DISCUSSION TITLE
Removal of Docker Containers

### DISCUSSION THREAD

**Discussion by fdefalco** [OHDSI/Achilles#536] [General]
General announcement on the removal of the docker container specifications from this repository. 

This repository is intended to contain an R package called Achilles that runs characterization analyses on a CDM standardized database.  After discussing with the prior maintainer it was determined that images to manage configuration and execution of R packages would be maintained separate from the package itself.  I'm fully supportive of having a way to enable the work people are trying to do with Achilles, and identify approaches or create repositories as necessary.  As it stands, the replacement solution is https://github.com/OHDSI/OHDSI-in-a-Box.

**Comment by chgl** [comment]
So the way forward would be to move the [Dockerfile](https://github.com/OHDSI/Achilles/blob/f23a2763c2eb073ec24faddc74b92e14a5224948/Dockerfile) (which needs to be modified to instead clone the Achilles repo on build) and the [docker-run](https://github.com/OHDSI/Achilles/blob/f23a2763c2eb073ec24faddc74b92e14a5224948/docker-run) to the OHDSI-in-a-box repo and setup GitHub Actions to build and push the image?

We could automate the process so the image is build everytime there's a new tag/release in this repo using eg. https://github.com/renovatebot/renovate - it's what I'm using over at https://github.com/chgl/kube-powertools to stay up-to-date on external releases.

**Reply by fdefalco** [reply]
Yes, something like that sounds like it would work and I am willing to setup whatever actions are required in this repo to notify things downstream.
